Make ChipsLayoutManger implement IFlexibleLayoutManager

magneticflux- opened the pull request:
Closes #43, closes #30
This allows users to use the ChipsLayoutManger with the FlexibleAdapter library.

magneticflux- committed:
Update build tools and Gradle Wrapper

magneticflux- committed:
Added Google repository and provided dependency on FlexibleAdapter library

magneticflux- committed:
Made ChipsLayoutManager implement IFlexibleLayoutManager

magneticflux- committed:
Fixed copy-paste error by aligning TopGravityModifier with BottomGravityModifier

tspoke commented:
@BelooS Any update ? When do you think this PR will be included ? I really need it too :)

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 638.63kb -> 491.37kb (23.06%)

/sample/src/main/res/drawable-nodpi/girl5.png -- 76.18kb -> 42.84kb (43.76%)
/sample/src/main/res/drawable-nodpi/girl2.png -- 50.27kb -> 30.33kb (39.66%)
/sample/src/main/res/drawable-nodpi/girl1.png -- 50.97kb -> 31.01kb (39.17%)
/sample/src/main/res/drawable-nodpi/girl4.png -- 39.90kb -> 25.01kb (37.32%)
/sample/src/main/res/drawable-nodpi/karl.png -- 21.75kb -> 14.47kb (33.46%)
/sample/src/main/res/drawable-nodpi/anonymous.png -- 69.98kb -> 47.93kb (31.5%)
/sample/src/main/res/drawable-nodpi/batman.png -- 34.01kb -> 24.93kb (26.7%)
/sample/src/main/res/drawable-nodpi/china_girl.png -- 42.64kb -> 38.35kb (10.04%)
/images/for_android.svg -- 5.15kb -> 4.67kb (9.46%)
/sample/src/main/res/mipmap-xxxhdpi/ic_launcher.png -- 10.24kb -> 9.45kb (7.73%)
/images/header.png -- 97.06kb -> 90.07kb (7.2%)
/sample/src/main/res/mipmap-xxhdpi/ic_launcher.png -- 7.54kb -> 7.04kb (6.66%)
/images/rtl_horizontal.png -- 61.23kb -> 57.56kb (5.99%)
/images/ltr_horizontal.png -- 61.50kb -> 57.92kb (5.82%)
/sample/src/main/res/mipmap-xhdpi/ic_launcher.png -- 4.73kb -> 4.46kb (5.76%)
/sample/src/main/res/mipmap-hdpi/ic_launcher.png -- 3.34kb -> 3.24kb (2.9%)
/sample/src/main/res/mipmap-mdpi/ic_launcher.png -- 2.15kb -> 2.10kb (2.4%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

magneticflux- committed:
Merge pull request #1 from magneticflux-/imgbot

[ImgBot] Optimize images
